src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -109,6 +109,7 @@ function BaseSelectionList<TItem extends ListItem>(
         const allOptions: TItem[] = [];
 
         const disabledOptionsIndexes: number[] = [];
+        const disabledArrowKeyOptionsIndexes: number[] = [];
```

Comment:
Can we add a comment explaining what these two arrays are for? Or add it to the comment above the flattenedSections function.

Reply:
Added the comment

Reply:
Conflicts solved and comment updated. Please check